README.md

```diff
@@ -0,0 +1,164 @@
+versions of the dependency it is compatible with, and this is a requirement for
+automerging. This is done in the package's `Project.toml` file, and is
+documented [here](https://julialang.github.io/Pkg.jl/v1/compatibility/) in the
+Julia package manager's documentation.
+
+Packages should at least declare the Julia version they are compatible with, by
+e.g.
+
+```toml
+[compat]
+julia = "1"
+```
+
+which declares that the package is compatible with all Julia versions from 1.0
+onwards, until but not including, a possible 2.0 in the future. The automerge
+bot will not merge packages that do not provide upper bounds for every package.
+This means that, e.g.
+
+```toml
+[compat]
+julia = ">= 1"
+```
+
+would not be sufficient. This is because such a statement declares that the
+version of the package being registered is compatible with every future version
+of Julia, but this is impossible to know at this stage. To claim compatibility
+with more than one breaking release of a package, add multiple entries such as
+
+```toml
+[compat]
+StaticArrays = "0.10, 0.11"
+```
+
+See [the `Pkg`
+documentation](https://julialang.github.io/Pkg.jl/v1/compatibility/) for more on
+how to specify `compat` entries.
+
+Note that standard libraries are currently coupled to the Julia version, and
+don't have separate version numbers. So while they are declared as dependencies,
+they don't need separate entries in the `compat` section from Julia itself.
+
+### Merge conflicts
+
+If the pull request against the General registry has merge conflicts with the
+General registry itself, the pull request cannot be merged. In this case, just
+retrigger the Registrator.jl bot (either add another comment with
+"@JuliaRegistrator register", or use the web interface), and it will update the
+pull request.
+
+### Julia Package naming conventions
+
+The guidelines for naming packages are listed
+[here](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1)
+in the package manager's documentation. If a new package's name does not adhere
+to those guidelines or could otherwise cause confusion, maintainers of the
+General registry or the Julia community might comment on the pull request to ask
+that the name be changed, or why the usual conventions were not used in this
+case. Additionally, the automerge bot will not merge pull requests corresponding
+to new packages whose names do not meet certain requirements. The package can
+still be added by manually merging the request, however, and updates to the
+package will still be automerged as usual.
+
+If you think that your package should not follow those conventions for some
+reason or another, just explain why. Otherwise, it is often a good idea to just
+rename the package-- it is more disruptive to do so after it is already
+registered, and sticking to the conventions makes it easier for users to
+navigate Julia's many varied packages.
+
+To rename the package from `OldName.jl` to `NewName`.jl,
```

Comment:
```suggestion
As long as the package is not yet registered; renamiing the package from `OldName.jl` to `NewName`.jl, is reasonably straght-forward:
```

Just so peole are really clear that these instructions  only apply to unregistered packages

Reply:
Sounds good, I added this clarification